Signals/Domain/MissingValuePolicy/FirstOrderMissingValuePolicy.cs

```diff
@@ -40,24 +49,30 @@ public class FirstOrderMissingValuePolicy<T> : MissingValuePolicy<T>
 
         private Quality GetMinQuality(Quality a, Quality b)
         {
-            // TODO should we add numbers to Quality? Those would probably go into Dto and suggest some impl...
             if (a == Quality.None || b == Quality.None)
                 return Quality.None;
-            return (Quality)Math.Max((int)a, (int)b);
+
+            return Enum
+                .GetValues(typeof(Quality))
+                .Cast<Quality>()
+                .Last(q => q == a || q == b);
```

Comment:
true, but this smells like one of those magic places nobody will ever find if he/she encounters error :D (imagine, somebody sorts enum alphabetically, and first order interpolation integration tests fail :D)
but yeah - for now - good 'nuf